jujugui/static/gui/src/app/app.js

```diff
@@ -1135,6 +1126,9 @@ YUI.add('juju-gui', function(Y) {
         // If the controller isn't ready yet then don't render anything.
         return;
       }
+      const getCreds = function() {
```

Comment:
This actually would fail b/c it's a func, and the component wants a string--so we're just passing in the username now.